CHANGES.rst

```diff
@@ -27,6 +27,11 @@ Release History
 - SqrtBeta distribution gives distribution of semantic pointer elements.
   (`#414 <https://github.com/nengo/nengo/issues/414>`_,
   `#430 <https://github.com/nengo/nengo/pull/430>`_)
+- Various improvements of unit testing infrastructure.
+  (`#648 <https://github.com/nengo/nengo/pull/648>`_,
+  `#629 <https://github.com/nengo/nengo/pull/629>`_,
+  `#647 <https://github.com/nengo/nengo/pull/647>`_,
+  `#653 <https://github.com/nengo/nengo/pull/653>`_)
```

Comment:
I'm fine with doing it either way. Maybe we should add another section for stuff which is more relevant to developer than end users?